File: app/src/main/java/cz/kutner/comicsdb/connector/service/SeriesService.java
package cz.kutner.comicsdb.connector.service;

import java.util.List;

import cz.kutner.comicsdb.model.Series;
import retrofit.http.GET;
import retrofit.http.Query;
import rx.Observable;

public interface SeriesService {
    @GET("/serielist.php")
    Observable<List<Series>> getSeriesList(@Query("str") int page);

    @GET("/search.php")
    Observable<List<Series>> searchSeries(@Query("searchfor") String keyword);
}

File: app/src/main/java/cz/kutner/comicsdb/connector/service/SeriesService.kt
package cz.kutner.comicsdb.connector.service

import cz.kutner.comicsdb.model.Series
import retrofit.http.GET
import retrofit.http.Query

interface SeriesService {
    @GET("/serielist.php")
    fun getSeriesList(@Query("str") page: Int): List<Series>

    @GET("/search.php")
    fun searchSeries(@Query("searchfor") keyword: String): List<Series>
}
